src/lib/sed.c

```diff
@@ -72,6 +74,7 @@ static struct opal_interface opal_if = {
 	.ownership_fn = sedopal_takeownership,
 	.revert_fn = sedopal_reverttper,
 	.activatelsp_fn = sedopal_activatelsp,
+	.revertsp_fn = sedopal_revertsp,
```

Comment:
Suggest to set it to NULL and handle like in other functions, e.g. 


int sed_ds_anybody_read(struct sed_device *dev, uint8_t *to, uint32_t size, uint32_t offset)
{
	if (curr_if->ds_anybody_read_fn == NULL)
		return -EOPNOTSUPP;

	return curr_if->ds_anybody_read_fn(dev, to, size, offset);
}

Then sed_ioctl.[ch] would not have to be changed

Reply:
With this being added, I would suggest removing the corresponding function in the sed_ioctl.c (bcoz, we aren't even reaching there).

Reply:
done